merlin-server/sql/merlin/functions/public/begin_merge.sql

```diff
@@ -266,26 +266,26 @@ begin
   union
   -- 'modify' against a 'modify' must be checked for equality first.
   select _merge_request_id, activity_id, name, tags,  source_scheduling_goal_id, created_at,
-         start_offset, type, arguments, metadata, anchor_id, anchored_to_start, 'none'
+         last_modified_by, start_offset, type, arguments, metadata, anchor_id, anchored_to_start, 'none'
   from (
     select activity_id, name, metadata.tag_ids_activity_directive(dd.activity_id, psa.snapshot_id) as tags,
-           source_scheduling_goal_id, created_at, start_offset, type, arguments, metadata, anchor_id, anchored_to_start
+           source_scheduling_goal_id, created_at, last_modified_by, start_offset, type, arguments, metadata, anchor_id, anchored_to_start
       from plan_snapshot_activities psa
       join diff_diff dd
         on dd.activity_id = psa.id
       where psa.snapshot_id = snapshot_id_supplying
         and (dd.change_type_receiving = 'modify' and dd.change_type_supplying = 'modify')
     intersect
     select activity_id, name, metadata.tag_ids_activity_directive(dd.activity_id, ad.plan_id) as tags,
-           source_scheduling_goal_id, created_at, start_offset, type, arguments, metadata, anchor_id, anchored_to_start
+           source_scheduling_goal_id, created_at, last_modified_by, start_offset, type, arguments, metadata, anchor_id, anchored_to_start
       from diff_diff dd
       join activity_directive ad
         on dd.activity_id = ad.id
       where ad.plan_id = plan_id_receiving
         and (dd.change_type_supplying = 'modify' and dd.change_type_receiving = 'modify')
   ) a;
```

Comment:
This is incorrect, as it is comparing the `last_modified_by` column. The code at the bottom of this comment is what I was referring to when I said "do the comparison, then join on `activity_directive` to get the `last_modified_by` column". 

Additionally, I'd like a new test in the `PlanCollaborationTests` that has a different user modify the activity on the plan and the branch, so that this bug cannot be accidentally reintroduced. Alternatively, updating `identicalModifyModifyResolvesAsNone` would be sufficient.

(Note that this has been split into its own `INSERT` statement for readability and should replace the entire highlighted block. I'd've put it as a suggestion, but GH seems to think you deleted lines in here somewhere and won't let me.)

```sql
  -- 'modify' against a 'modify' must be checked for equality first.
  with false_modify as (
  	select activity_id, name, metadata.tag_ids_activity_directive(dd.activity_id, psa.snapshot_id) as tags,
           source_scheduling_goal_id, created_at, start_offset, type, arguments, metadata, anchor_id, anchored_to_start
      from plan_snapshot_activities psa
      join diff_diff dd
        on dd.activity_id = psa.id
      where psa.snapshot_id = snapshot_id_supplying
        and (dd.change_type_receiving = 'modify' and dd.change_type_supplying = 'modify')
    intersect
    select activity_id, name, metadata.tag_ids_activity_directive(dd.activity_id, ad.plan_id) as tags,
           source_scheduling_goal_id, created_at, start_offset, type, arguments, metadata, anchor_id, anchored_to_start
      from diff_diff dd
      join activity_directive ad
        on dd.activity_id = ad.id
      where ad.plan_id = plan_id_receiving
        and (dd.change_type_supplying = 'modify' and dd.change_type_receiving = 'modify'))
  insert into merge_staging_area (
    merge_request_id, activity_id, name, tags, source_scheduling_goal_id, created_at, last_modified_by,
    start_offset, type, arguments, metadata, anchor_id, anchored_to_start, change_type)
  select _merge_request_id, ad.id, ad.name, tags,  ad.source_scheduling_goal_id, ad.created_at,
         ad.last_modified_by, ad.start_offset, ad.type, ad.arguments, ad.metadata, ad.anchor_id, ad.anchored_to_start, 'none'
  from false_modify fm left join activity_directive ad on (ad.plan_id, ad.id) = (plan_id_receiving, fm.activity_id);
```